scripts/superdesk-monitoring/aggregate.js

```diff
@@ -381,6 +381,13 @@
             });
         };
 
+        this.reload = function() {
+            $timeout(function() {
+                self.search(self.searchQuery);
+            }, 1, false);
+            self.search();
```

Comment:
Please consider this solution instead: https://github.com/eozan/superdesk-client-core/pull/3